CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,9 @@
+# Code of Conduct
+
+Please note that all interactions related to Junction are covered by
+the [PSF Code of Conduct](https://www.python.org/psf/codeofconduct/),
```

Comment:
Shouldn't this be the PyCon India CoC?

Reply:
PyCon India CoC is geared more towards a conference-only CoC, given the language.

PSF’s CoC is much more relaxed and general.